collect.py

```diff
@@ -164,6 +164,9 @@ def do_something(driver, elem_attributes=None):
         elif input_type == 'search':
             elem.clear()
             elem.send_keys('quick search')
+        elif input_type == 'color':
+            elem.click()
+            driver.execute_script("arguments[0].setAttribute('value', '#FF0000')", elem)
```

Comment:
You can use `arguments[0].value = '#FF0000'`.
Or even better, use the Selenium API to do it

Reply:
I tried to find a set_attribute function in Selenium WebDriver and WebElement API but it wasn't present. So I went for javascript's setAttribute.
The `.value` notation looks cleaner to me, will try it :)
I'll replace the `.click()` with an ActionChain, so I move to the element and set the value.

Reply:
If you use `.value`, you don't need to do anything else before